charts/seed-controlplane/charts/etcd/templates/configmap-etcd-bootstrap.yaml

```diff
@@ -9,8 +9,10 @@ metadata:
 data:
   bootstrap.sh: |-
     #!/bin/sh
-    if [ -d /var/etcd/data/member ]; then
+    if [ ! -d /var/etcd/data/new.etcd ]; then
         mkdir /var/etcd/data/new.etcd
+    fi
+    if [ -d /var/etcd/data/member ]; then
```

Comment:
```suggestion
    else
```

Reply:
From @georgekuruvillak:

> In case of existing shoot deployments, we will need to mkdir new.etcd as well as mv existing data directory inside new.etcd.
> In case of new clusters as there won't be any member directory we need to do just mkdir new.etcd
> So if we have else here, in case of existing cluster the new.etcd directory will get created but member directory won't be moved.